### PR TITLE
Minor changes to the marshal leverguns

### PR DESCRIPTION
**Pull request by maikilangiolo**
## About The Pull Request
1. they are no longer blacklisted, but the obrezes are
2. cleaned up a bit of the code, extra FS serial var not needed, they were all children of the levergun rifle
3. buffed the lever gun ammo capacity. it only had 6 shells, less than the mateba, and its a whole lever gun rifle. Holds 10, which is an entire .40 bullet pile for easy reload (like a bolt gun)
4. slightly lowered the levergun shotgun ammo capacity, since the gun it is based on held 7 shells, and the gun hits hard.

## Why It's Good For The Game
Every other special role gun was not blacklisted, powerful antag only guns, and these two are not exceptions. There is no reason for the special treatment.
The rifle was also buffed in ammo capacity because it was too low and made the gun weak, as .40 revolvers already held 6 and even 7 shells, so its an improvement.

## Changelog
:cl:
del: removed anomalous marshal lever guns blacklist
tweak: changed lever rifle ammo 6 -> 10
tweak: changed lever shotgun ammo 9 -> 7
fix: removed unnecessary FS serial vars since they were children and inherited the serial already
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/battle_rifle/levergun.dm [hunk 1]
```diff
@@ -76,3 +74,4 @@
 	saw_off = TRUE
 	can_dual = TRUE
 	twohanded = FALSE
+	spawn_blacklisted = TRUE
```
Comment (zombiemimic) [inline code/modules/projectiles/guns/projectile/battle_rifle/levergun.dm:77]: newline

**Comment by maikilangiolo** [comment]
> You got no permission to unblackiist the marshal gun's

I don't need to. It's open source, and if you use open source code (the one you made the gun with), you cannot stop people from using it and modifying it,  same if you release your code as open source. When you write your eris codebase from the ground up, well, you can do what you want. 
You didn't sprite the gun, even.

**Comment by thedarkcrusader** [comment]
> > You got no permission to unblackiist the marshal gun's
> 
> I don't need to. It's open source, and if you use open source code (the one you made the gun with), you cannot stop people from using it and modifying it, same if you release your code as open source. When you write your eris codebase from the ground up, well, you can do what you want. You didn't sprite the gun, even.

There are no problems with you editing it's code or anything for that matter it is clearly intended that the marshal lever rifles remain within the marshal uplink only tools that I specifically got permission from Firefox to do

**Comment by TheShown911** [comment]
No.

**Review by Firefox13 — APPROVED** [review]
I talked with Maik about it being not spawnable and we came to a conclusion that leverguns are more usable as a proof that person is a marshall than a marshall's badge itself. Which isn't good.
Besides, only few, very "broken" (RPG) guns are blacklisted and these two (four if you count sawn-offs) guns certainly do not fall into that power range.

**Review by Firefox13 — COMMENTED** [review]
Just code (and sprite) a frame for it, mmkay?

**Comment by maikilangiolo** [comment]
> Just code (and sprite) a frame for it, mmkay?

image
done (zamn!!! codersprites!!!)
also if you see more number changes, I am trying to avoid merge conflict with #7563 (i dont know how to rebase)

**Review by Firefox13 — APPROVED** [review]

**Comment by hyperioo** [comment]
@maikilangiolo Conflict in crafts.dmi

**Comment by maikilangiolo** [comment]
Why can't I resolve the conflict here guh

**Comment by maikilangiolo** [comment]
Oh no, it's a .dmi conflict. I can't fix it now

**Comment by maikilangiolo** [comment]
I HATE DMI CONFLICTS REEEEEEEEEEEEE

**Pull request closed without merging**